### PR TITLE
Document adding pages

### PR DESCRIPTION
**Pull request by mstriemer**
One commit with some documentation, one with the page that I made while writing the documentation. That might be weird that the page already exists so you can't really follow it but we'll probably want something like this so I included it.

### CODE REVIEW

**Comment by mstriemer** [comment]
I noticed that we could abstract the entities reducers like addons and users away entirely. It was really just copy/paste. The best would be to have an `entities` reducer that nests all of the normlizr entities under `entities`, so this would impact other parts of the code.

It might be nice to add redux-thunk even though we're using redux-async-connect. I think we'd like to be using thunk instead as it seems more popular, we just need to figure out the server-side rendering bit. Using thunk would allow us to move the `loadEntities` dispatch into middleware which would clean things up a little.

**Comment by muffinresearch** [comment]
Good work on starting this now 👍.  I know we'd said we wanted some docs but it would have been easy to end-up not getting around to it.

File: docs/adding-a-page.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+src/
+  <app-name>/
+    components/
+      MyComponent/
+        index.js
+        styles.scss
+    containers/
+      MyContainer/
+        index.js
+        styles.scss
```
Comment (muffinresearch) [inline docs/adding-a-page.md]: CSS is currently in its own directory as opposed to being like this next to the JS.

Comment (mstriemer) [inline docs/adding-a-page.md]: I've been putting my CSS next to the JS it belongs to. We should pick one way though. Maybe we should make a list of things we haven't been consistent on and go through it and enforce stuff with eslint/tests.

**Comment by mstriemer** [comment]
Andy suggested putting these on Read the Docs which I think is a good idea. I'll figure out how to get them up there in a bit.

**Pull request merged**